ci/lint.sh

```diff
@@ -3,6 +3,13 @@
 # SPDX-FileCopyrightText: Red Hat, Inc.
 # SPDX-License-Identifier: GPL-2.0-or-later
 
+# Workaround to avoid this warning:
+#   fatal: detected dubious ownership in repository at '/dir'
+git config --global --add safe.directory "$(pwd)"
```

Comment:
This is another thing I think we should do only inside container env, so I propose:
```bash
[ -d /venv ] && {
    git config ...    
    source /venv/bin/activate
}
```